Add WAL failover demonstration video (#19418)
cockroachdb/docs: src/current/v24.3/wal-failover.md

However, the WAL failback operation will not cascade back until **all drives are available** - that is, if store _A_'s disk unstalls while store _B_ is still stalled, store _C_ will not failback to store _A_ until _B_ also becomes available again. In other words, _C_ must failback to _B_, which must then failback to _A_.

## Video demo: WAL failover

> To explain WAL Failover in CockroachDB and demonstrate what happens when you enable vs disable it, join our Technical Evangelist at Cockroach Labs Rob Reid for this in-depth video:

Reviewer: I suggest simplifying the text to make it more consistent with our other video examples:

```suggestion
For a demo of WAL Failover in CockroachDB and what happens when you enable or disable it, play the following video:
```

Author: ✅
